superset/assets/src/dashboard/deprecated/v1/components/Header.jsx

```diff
@@ -43,25 +43,11 @@ class Header extends React.PureComponent {
       showV2PromptModal: props.dashboard.promptV2Conversion,
     };
     this.toggleShowV2PromptModal = this.toggleShowV2PromptModal.bind(this);
-    this.handleConvertToV2 = this.handleConvertToV2.bind(this);
+    this.handleConvertToV2 = props.handleConvertToV2.bind(this);
```

Comment:
why do you need to bind this? it's already bound at the dashboard level right?

Reply:
`handleConvertToV2` is refactored out into Dashboard component

Reply:
right but you don't need to bind a function that comes from props.

Reply:
yes.... fixed.